### ISSUE TITLE
Non-existing outfitting modules via EDDN

### ISSUE THREAD

**Issue by themroc5** [Duke-Jones/ED-IBE#8]
We discussed this before. EDIBE sends not existing modules via EDDN. Please remove the auto guessing and use the [correct mapping](https://github.com/EDCD/FDevIDs).

Sample EDDN message: https://ross.eddb.io/eddn/log/7814558

Non existing modules:

```
{
    "category": "hardpoint",
     "rating": "D",
     "mount": "Gimballed",
     "name": "Burst Laser",
     "class": "4"
},
{
    "category": "hardpoint",
     "rating": "D",
     "mount": "Fixed",
     "name": "Multi-Cannon",
     "class": "3"
},
{
    "category": "hardpoint",
     "rating": "D",
     "mount": "Gimballed",
     "name": "Pulse Laser",
     "class": "4"
},
```

Thanks :)

**Comment by themroc5** [comment]
I consider this as critical. It has been a month and nothing changed. Still EDDN is being polluted with wrong entries. When is it gonna be fixed?

**Comment by themroc5** [comment]
EDDB now blocks your outfitting messages :)

https://ross.eddb.io/eddn/log?onlyError=1

**Comment by Duke-Jones** [comment]
should be fixed in v0.4.0
(https://github.com/Duke-Jones/ED-IBE/tree/v0.4.0)

you may remove ED-IBE (or v0.4.0 and higher) from the blacklist ?

EDIT: I'm sorry for the long duration to fix this.  I read your comment from 25 Jul 10 minutes before for the first time. I never realizied that this problem was so critical. Now I'm back on development :)

**Comment by Duke-Jones** [comment]
still working on this.
OtisB told me there are still the wrong names at least in the shipyard messages.
I' switching to "commodity v3"/"outfittingv2"/"shipyardv2" now....

**Comment by Duke-Jones** [comment]
since 62cdf59a5679c1f8e9ba6229c3987c20139b467b ED-IBE uses 
- commodity v3
- outfitting v2
- shipyard v2

so the problem should be fully fixed with the next release v0.4.1

**Comment by Duke-Jones** [comment]
released v0.4.1, closed
